features/testbot/ferry.feature

```diff
@@ -30,15 +30,15 @@ Feature: Testbot - Handle ferry routes
             | fg    |         | ferry | 0:10     |
             | jk    |         | ferry | 1:00     |
             | no    |         | ferry | 24:00    |
-            | rs    |         | ferry | 96:00    |
+            | rs    |         | ferry | 72:00    |
```

Comment:
One segment with clipping like  `| vw    |         | ferry | 96:00    |`  is still needed